PyPDF2/generic/_fit.py

```diff
@@ -0,0 +1,129 @@
+from typing import Any, Optional, Tuple, Union
+
+
+class Fit:
+    def __init__(
+        self, fit_type: str, fit_args: Tuple[Union[None, float, Any], ...] = tuple()
+    ):
+        from ._base import NameObject, NullObject, NumberObject
+
+        self.fit_type = NameObject(fit_type)
+        self.fit_args = [
+            NullObject() if a is None or isinstance(a, NullObject) else NumberObject(a)
```

Comment:
I would say that NumberObject should be replaced  by FloatObject

Reply:
Thank you - done :-)